### PR TITLE
Server uptime monitoring blog

### PR DESCRIPTION
**Pull request by sashwathn**
Please review and approve.

### CODE REVIEW

File: blog/2023-05-02-server-uptime-monitoring-why-do-we-need-it/index.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+<!-- truncate -->
+
+## Why Server uptime monitoring is important?
+
+Servers are crucial to the operation of many applications and services, making their uptime a critical aspect of overall system performance. Here are some reasons why server uptime monitoring is important:
+
+
+- **Availability**
+    Uptime monitoring can show whether a server is consistently available or experiences frequent downtimes. High uptime indicates the server is reliable and available for users, while low uptime may signal potential issues that need to be addressed.
```
Comment (cakrit) [inline blog/2023-05-02-server-uptime-monitoring-why-do-we-need-it/index.md:22]:
```suggestion
  Uptime monitoring can show whether a server is consistently available or experiences frequent downtimes. High uptime indicates the server is reliable and available for users, while low uptime may signal potential issues that need to be addressed.
```

File: blog/2023-05-02-server-uptime-monitoring-why-do-we-need-it/index.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+image: ./img/stacked-netdata.png
+---
+
+![stacked-netdata](./img/stacked-netdata.png)
+
+Server uptime monitoring tracks the availability and reliability of servers within your infrastructure.
+
+<!-- truncate -->
+
+## Why Server uptime monitoring is important?
```
Comment (cakrit) [inline blog/2023-05-02-server-uptime-monitoring-why-do-we-need-it/index.md:16]:
```suggestion
## Why is Server Uptime Monitoring Important?
```

**Review by cakrit — CHANGES_REQUESTED** [review]
Two more comments:
- The lists don't appear properly in the HTML. You've done proper markdown, but the text on the blog will appear next to the bold items. @andrewm4894 can check if it's something particular to Docusaurus, or we can just go back to using ":" after the bold.
- Please at the bottom add a small section about Netdata. I have a chart screenshot here you can add

image

**Comment by sashwathn** [comment]
@cakrit : All changes made. Approve when you get a chance.

**Review by cakrit — APPROVED** [review]

**Pull request merged**